utils/spdx/src/main/kotlin/SpdxExpression.kt

```diff
@@ -410,7 +410,10 @@ class SpdxLicenseWithExceptionExpression(
     val exception: String
 ) : SpdxSingleLicenseExpression() {
     companion object {
-        private val EXCEPTION_STRING_PATTERN = Regex("\\b(exception|additional-terms)\\b", RegexOption.IGNORE_CASE)
+        private val EXCEPTION_STRING_PATTERN = Regex(
```

Comment:
Reading the license text for `https://scancode-licensedb.aboutcode.org/ecma-no-patent.html`, I wonder if the text is really an exception. The only reason why it might be an exception to me is due to the reference to "this license". However, do we maybe have possibility of a legal persons input?

If it isn't an exception, the the fix would be:
1. A license finding curation
2. Report an issue to ScanCode to report the finding using and `AND` instead of a `WITH` ?

Reply:
https://scancode-licensedb.aboutcode.org/ecma-no-patent.yml has `is_exception: yes`.